### ISSUE TITLE
Are there plans to port the workflow to DSL2?

### ISSUE THREAD

**Issue by scontente** [nf-core/dualrnaseq#88]
Currently, this pipeline will not run on the latest version of Nextflow. The error message " Nextflow DSL1 is no longer supported — Update your script to DSL2, or use Nextflow 22.10.x or earlier", appears.

**Comment by reganhayward** [comment]
Hi there,
Thanks for your message. Yes, this is currently being worked on by a number of people, you can see this from the activity in the dev branch. As you can imagine, there is a considerable amount of work required, and hopefully this will be completed soon!

**Comment by scontente** [comment]
Thank you!  Understand how much work goes into an effort like this. On Jun 26, 2023, at 3:11 AM, reganhayward ***@***.***> wrote:﻿
Hi there,
Thanks for your message. Yes, this is currently being worked on by a number of people, you can see this from the activity in the dev branch. As you can imagine, there is a considerable amount of work required, and hopefully this will be completed soon!

**Comment by IsabelFE** [comment]
Are there any updates on the DSL2 version of this pipeline?

**Comment by reganhayward** [comment]
Yes - being worked on currently, but only as a single person. I'm hesitant to give an exact timeframe, but should be soon!

**Comment by IsabelFE** [comment]
@reganhayward, thanks for your work!!
